test/Cardano/Address/Style/ShelleySpec.hs

```diff
@@ -62,7 +64,8 @@ prop_publicChildKeyDerivation
 prop_publicChildKeyDerivation (mw, sndFactor) cc ix =
     addrXPub1 === addrXPub2
   where
-    rootXPrv = genMasterKeyFromMnemonic mw sndFactor :: Shelley 'RootK XPrv
+    rootXPrv =
+        genMasterKeyFromMnemonic mw (maybeMnemonicToBytes sndFactor) :: Shelley 'RootK XPrv
```

Comment:
What about changing the argument of the property instead to be `ScrubbedBytes` instead of `Maybe SomeMnemonic` :) ?

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/20/commits/b97b25b7bbf9c5f2130b016a8e88457325063549